lib/ak/updater.rb

```diff
@@ -0,0 +1,30 @@
+# Monkey Patch!
+# To be moved and spec'd at https://github.com/EricBoersma/actionkit_connector
+module ActionKitConnector
```

Comment:
This code is adding to https://github.com/EricBoersma/actionkit_connector - I want to have a working build on staging before updating Eric's gem and having a new version cut. Does that make sense?